plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -1151,6 +1151,18 @@ public void testInsertRowConcurrently()
         abort("Test fails with a timeout sometimes and is flaky");
     }
 
+    @Test
+    public void testArrayDataType()
```

Comment:
Type mapping test should go to `BaseBigQueryTypeMapping`.

Reply:
moved to BaseBigQueryTypeMapping